doc/WAW_dialog_doc.txt

```diff
@@ -194,14 +194,16 @@
 	**************************************************
 	** <goto>
 	**************************************************
-	Represents a jump to an another node.
+	Represents a jump to an another node or another type of next step.
```

Comment:
typo: to an another -> to another (== an other)

Reply:
Thanks for spotting it, fixed.